controllers/storageclaim_controller.go

```diff
@@ -651,6 +673,70 @@ func (r *StorageClaimReconciler) hasVolumeSnapshotContents() (bool, error) {
 	return false, nil
 }
 
+func (r *StorageClaimReconciler) getCephDriverVolumeGroupSnapshotClass(driverType string,
```

Comment:
Can we please move this function near the `getCephDriverVolumeSnapshotClass` function? both of them have similar concerns and I would prefer to have both of them at a similar location in the file